src/main/scala/shine/OpenCL/FlagPrivateArrayLoops.scala

```diff
@@ -28,6 +28,9 @@ object FlagPrivateArrayLoops {
       extends VisitAndRebuild.Visitor
     {
       override def phrase[T <: PhraseType](p: Phrase[T]): Result[Phrase[T]] = p match {
+        // FIXME: we would like this primitive to be eliminated before this pass?
```

Comment:
Can you also create an issue for this one?

Reply:
I added a comment in https://github.com/rise-lang/shine/issues/22